src/applications/vaos/tests/list/video-appointments.unit.spec.jsx

```diff
@@ -128,6 +128,65 @@ describe('VAOS integration: upcoming video appointments', () => {
     );
   });
 
+  it('should show active link less than 30 minutes in the future', async () => {
+    const appointment = getVideoAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      facilityId: '983',
+      clinicId: null,
+      startDate: moment()
+        .add(20, 'minutes')
+        .format(),
+    };
+    appointment.attributes.vvsAppointments[0] = {
+      ...appointment.attributes.vvsAppointments[0],
+      dateTime: moment()
+        .add(20, 'minutes')
+        .format(),
+      bookingNotes: 'Some random note',
```

Comment:
Could probably take this out. I've been trying to remove mock data that the test being run doesn't rely on.